packages/frameworks-sveltekit/src/lib/client.ts

```diff
@@ -31,6 +32,40 @@ export type SignInAuthorizationParams =
   | Record<string, string>
   | URLSearchParams
 
+/**
+ * Fetch webauthn options from server and prompt user for authentication or registration.
+ * Returns either the completed WebAuthn response or an error request.
+ *
+ * @param providerId provider ID
+ * @param options SignInOptions
+ * @returns WebAuthn response or error
+ */
+async function webAuthnOptions(providerId: string, options?: SignInOptions) {
+  const { startAuthentication, startRegistration } = await import(
+    "@simplewebauthn/browser"
+  )
```

Comment:
I still think we should follow the same pattern as for `next-auth` for consistency. `@auth/sveltekit/webautn` for now. Could you do a bundle analysis and verify that if I don't use WebAuthn, my client-side bundle won't include this import?

Reply:
Okay, did the same thing as with `next-auth` for now. A new `@auth/sveltekit/webauthn` exported `signIn`. It also still has the logic for the other signin methods so users can only use that 1 and don't need to juggle 2 if they have webauthn + oauth (for example) providers in their app, just like in `next-auth` :+1: